lumicks/pylake/detail/confocal.py

```diff
@@ -285,14 +295,15 @@ def timestamps(self) -> np.ndarray:
     def pixelsize_um(self):
         """Returns a `List` of axes dimensions in um. The length of the list corresponds to the
         number of scan axes."""
-        return [axes["pixel size (nm)"] / 1000 for axes in self._ordered_axes()]
+        return self._pixelsize_factory(self)
 
     @property
     def size_um(self):
         """Returns a `List` of scan sizes in um along axes. The length of the list corresponds to
         the number of scan axes."""
         return [
-            axes["pixel size (nm)"] * axes["num of pixels"] / 1000 for axes in self._ordered_axes()
+            pixel_size * num_pixels
+            for pixel_size, num_pixels in zip(self.pixelsize_um, self._num_pixels)
         ]
```

Comment:
```suggestion
        return map(lambda pixel_size, num_pixels: pixel_size * num_pixels, self.pixelsize_um, self._num_pixels)
```
should work as well and IMO expresses intent better than first zipping and then looping over the two pixel iterables.

Reply:
Sounds good! Should still actually explicitly construct the list though, since map returns a map object (iterator).

I'll change it to 
```python
        return list(
            map(
                lambda pixel_size, num_pixels: pixel_size * num_pixels,
                self.pixelsize_um,
                self._num_pixels,
            )
        )
```